### DISCUSSION TITLE
Goods Received Note (GRN)

### DISCUSSION THREAD

**Discussion by sreeisalso** [saleor/saleor#6932] [Ideas]
What is a Goods Received Note?
Goods Received Note is a record of goods received from suppliers, and the record is shown as a proof that ordered products had been received. Moreover, the record is used by the buyer for comparing the number of goods ordered to the ones delivered.

GRN plays an important role among various departments for several companies. Primarily, factory/store, procurement and finance/accounting departments use the record for stock updates and the payment of goods procured.

**Comment by karolkielecki** [comment]
@sreeisalso Thanks for your suggestion! I've passed it to the team. Cheers!

**Comment by blueyo** [comment]
@sreeisalso I think GRN is a warehouse activity part of the WMS or ERP. I do not see that being part of the Saleor workflow. It would be nice though to have an Saleor plugin for use cases when there is a ERP/WMS that have received the products and wants to update Saleor to make the product published/Not published ready to be sold for customers. That I think would be achievable with the existing graphql mutation to control published/Not published.
